client/package.json

```diff
@@ -17,6 +17,7 @@
     "babel-preset-react-app": "1.0.0",
     "babel-preset-stage-1": "6.24.1",
     "case-sensitive-paths-webpack-plugin": "1.1.4",
+    "chai": "^4.1.2",
```

Comment:
Shouldn't the new dependencies be frozen at a specific version?

Reply:
Done.